### ISSUE TITLE
Turn off error: "Cannot write to INI file"

### ISSUE THREAD

**Issue by techhazard** [sabnzbd/sabnzbd#2522]
I'm running sabnzbd under NixOS and the TL;DR of that is that the config file is readonly.
Therefore it is expected that sabnzbd itself can't write to the config file.

However, it is annoying to see the errors pop up almost every second: "a few seconds ago: Cannot write to INI file"
Is there a way to turn off this error?

Secondary question: Why does sabnzbd want to write to the config file even when I'm not actively changing the settings?

**Comment by sanderjo** [comment]
> Why does sabnzbd want to write to the config file even when I'm not actively changing the settings?

Indeed. If you have your SAB GUI closed, then I can only see SABnzbd.py each 30 seconds:

```
            # Save config (if needed)
`            config.save_config()`
```

And indeed the code only kicks in when Change or ... Forced:

https://github.com/sabnzbd/sabnzbd/blob/develop/sabnzbd/config.py#L929

**Comment by thezoggy** [comment]
what if you set the config only variable `config_lock` to 1 (which forbids access to config)? 
https://sabnzbd.org/wiki/configuration/3.7/special

**Comment by techhazard** [comment]
@thezoggy That kinda helped, but I now get `Configuration locked, cannot save settings` every 30 seconds or so :joy: 
[screenshot](https://imgur.com/VV5fZ3G)

**Comment by techhazard** [comment]
Seeing the code in `SABnzbd.py`, I don't think that's configurable.

**Comment by techhazard** [comment]
Found it!
I was missing a block of categories at the end:
```ini
[categories]
[[*]]
name = *
order = 0
pp = 3
script = None
dir = ""
newzbin = ""
priority = 0
[[movies]]
name = movies
order = 0
pp = ""
script = Default
dir = ""
newzbin = ""
priority = -100
[[tv]]
name = tv
order = 0
pp = ""
script = Default
dir = ""
newzbin = ""
priority = -100
[[audio]]
name = audio
order = 0
pp = ""
script = Default
dir = ""
newzbin = ""
priority = -100
[[software]]
name = software
order = 0
pp = ""
script = Default
dir = ""
newzbin = ""
priority = -100
```
